modules/dd-trace-cpp/0.2.0/MODULE.bazel

```diff
@@ -0,0 +1,21 @@
+module(
+    name = "dd-trace-cpp",
+    version = "0.2.0",
+    compatibility_level = 1,
+)
+
+bazel_dep(
+    name = "bazel_skylib",
+    version = "1.2.1",
+)
+bazel_dep(
+    name = "rules_cc",
+    version = "0.0.9",
+)
+# -- bazel_dep definitions -- #
+
+non_module_dependencies = use_extension("//:extensions.bzl", "non_module_dependencies")
+use_repo(
+    non_module_dependencies,
+    "com_google_absl",
```

Comment:
This is available as a module `abseil-cpp`, please use that instead to avoid linker issues.

Reply:
Can you please already make the change here?